index.html

```diff
@@ -669,6 +669,30 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ========= Card 12 END =========  -->
+        <div class="card">
+          <p class="name">Vera</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">tschanzvera</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">tschanzvera</a>
+          </p>
+          <p class="about">Student</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+              </li>
```

Comment:
You should also add any useful resources inside the `href` that could help the fellow contrbutors. It could be any website URL which provides tutorials, documentations, resources etc.,

Reply:
Removed resources

Reply:
Hi @tschanzvera 
Please do note that adding few resources is a good thing, as it could help the fellow contributors. We just noticed that you removed the `resources` section itself, which is not recommended. In case you have no idea of what to add, you could add any resources/websites that you visit to resolve any coding related problems. 

We would like to have your contribution to get merged soon! Add few resources and commit back to make your PR merged!

Reply:
Thank you for the advice I added some resources now